.github/renovate.json5

```diff
@@ -14,6 +14,8 @@
     "d3-scale-chromatic", // we should bump this once we move to esm modules
     "eslint", // wait until `eslint-plugin-react-hooks>4.2.0` is released
     "globby", // we should bump this once we move to esm modules
+    "less", // update this together with "less-loader" when we move to webpack5 in grafana-toolkit
+    "less-loader", // update this together with "less" when we move to webpack5 in grafana-toolkit
```

Comment:
i think we should add this to the specific ignores for `grafana-toolkit` on line 33. that way if we use it anywhere else it can still be updated separately.